lieer/remote.py

```diff
@@ -183,11 +184,35 @@ def _cb (rid, resp, excep):
         i += 1
 
       try:
+        # we wait if there is a user_rate_delay
+        if user_rate_delay > 0:
+          if user_rate_ok > 10:
+            # gradually reduce if we had 10 ok batches
+            if user_rate_delay == 2:
```

Comment:
Why do you need to special case 2? It would probably be a bit simpler to just use

user_rate_delay = user_rate_delay // 2